src/Makefile

```diff
@@ -4,7 +4,7 @@ KERNELVERSION ?= $(shell uname -r)
 KDIR := /lib/modules/$(KERNELVERSION)/build
 PWD := $(shell pwd)
 INSTALL_MOD_DIR ?= extra
-EXTRA_CFLAGS := -g
+ccflags-y:= -g
```

Comment:
What's with this change?

Reply:
`EXTRA_*FLAGS` has been deprecated in favor of `*flags-y` for Kbuild.

[KBuild Makefile documentation: Section 3.7](https://www.kernel.org/doc/Documentation/kbuild/makefiles.txt)